includes/functions.php

```diff
@@ -2549,100 +1369,6 @@ function tsml_date_localised($format, $timestamp = null)
     return $datetime->format($format);
 }
 
-/**
- * Checks a list of imported meetings for necessary import changes / updates
- *
- * @param array  $feed_meetings                array of feed meetings to test for changes
- * @param string $data_source_url              feed source url
- * @param int    $data_source_parent_region_id feed parent region id
- * @return array[array, array, array]
- *       $import_meetings    array of imported meetings that need to be processes
- *       $delete_meeting_ids array of existing meeting post id's to delete
- *       $change_log         array of changes to existing
- */
```

Comment:
Since you're moving this one, maybe move this to `functions_import.php` and rename to `tsml_import_get_changed_meetings()` or something? Since it's very import related.

Reply:
good call